pwiz_tools/Shared/Common/SystemUtil/ProcessRunner.cs

```diff
@@ -203,5 +204,15 @@ public IEnumerable<string> MessageLog()
         {
             return _messageLog;
         }
+

```

Comment:
Feels like this should go into a new class named "ProcessEx". It is not really appropriate here. This class is about running another process not for asking questions about the currently running process.

Reply:
OK, will do.